meta-luneos/recipes-multimedia/pulseaudio/pulseaudio-modules-droid_git.bb

```diff
@@ -6,15 +6,14 @@ DEPENDS += "pulseaudio libhybris virtual/android-headers dbus udev"
 
 PACKAGE_ARCH = "${MACHINE_ARCH}"
 
-PV = "10.0.59+gitr${SRCPV}"
-SRCREV = "d53995823d335a06a25b98d9a7ecd25f06afbcea"
+PV = "10.0.73+gitr${SRCPV}"
```

Comment:
Shouldn't be 11.1.73 as per https://github.com/mer-hybris/pulseaudio-modules-droid/releases? We'd need to update the FILES_PN as well then

Reply:
Ah, well, it's the package release, and as long as we are still on PA 10.0, I find it more logical to stick to this major number...